### PR TITLE
printable blockie image

### PR DESCRIPTION
**Pull request by hsurf22**
Add printable blockie image

### CODE REVIEW

**Comment by gesseekur** [comment]
is there a way to make the blockie both printable and display? Instead of using duplicate code

**Review by rsteubs — APPROVED** [review]

File: src/components/MewBlockie/MewBlockie.vue [hunk 1]
```diff
@@ -34,7 +31,7 @@ export default {
      */
     address: {
       type: String,
-      default: ''
+      default: '0xAa682624a57b0Ac4215BeCa5126b8C2bD61609fF'
```
[the remark this answers is not in the document]
Comment (hsurf22) [reply src/components/MewBlockie/MewBlockie.vue]: fixed

**Comment by gesseekur** [comment]
Screen Shot 2021-08-31 at 4 26 48 PM
this is what i see on storybook

**Comment by hsurf22** [comment]
> Screen Shot 2021-08-31 at 4 26 48 PM
> 
> this is what i see on storybook

Inset shadow position fixed

**Comment by gesseekur** [comment]
@hsurf22 what does the flat prop do / mean ?

**Comment by hsurf22** [comment]
> @hsurf22 what does the flat prop do / mean ?

<div v-if="!flat" class="inset-shadow"></div>
It removes inset shadow,
and also it makes blockie printable.

**Review by gamalielhere — APPROVED** [review]

**Pull request merged**